metrics/Score Metric.ipynb

```diff
@@ -500,31 +537,96 @@
-   "execution_count": 6,
+   "execution_count": 23,
    "metadata": {},
    "outputs": [
     {
      "name": "stdout",
      "output_type": "stream",
      "text": [
-      "OSSF scores saved to ./data/ossf-scores/2024-06-26.csv\n"
+      "\n",
+      "No repositories *improved* by 10% or more.\n",
+      "\n",
+      "Repositories worsened by 10% or more:\n"
      ]
+    },
+    {
+     "data": {
+      "text/html": [
+       "<div>\n",
+       "<style scoped>\n",
+       "    .dataframe tbody tr th:only-of-type {\n",
+       "        vertical-align: middle;\n",
+       "    }\n",
+       "\n",
+       "    .dataframe tbody tr th {\n",
+       "        vertical-align: top;\n",
+       "    }\n",
+       "\n",
+       "    .dataframe thead th {\n",
+       "        text-align: right;\n",
+       "    }\n",
+       "</style>\n",
+       "<table border=\"1\" class=\"dataframe\">\n",
+       "  <thead>\n",
+       "    <tr style=\"text-align: right;\">\n",
+       "      <th></th>\n",
+       "      <th>Repository</th>\n",
+       "      <th>Previous Score</th>\n",
+       "      <th>Current Score</th>\n",
+       "      <th>Percentage Change</th>\n",
+       "    </tr>\n",
+       "  </thead>\n",
+       "  <tbody>\n",
+       "    <tr>\n",
+       "      <th>0</th>\n",
+       "      <td>Ed-Fi-Exchange-OSS/LMS-Toolkit</td>\n",
+       "      <td>5.9</td>\n",
+       "      <td>5.3</td>\n",
+       "      <td>-10.169492</td>\n",
+       "    </tr>\n",
+       "  </tbody>\n",
+       "</table>\n",
+       "</div>"
+      ],
+      "text/plain": [
+       "                       Repository  Previous Score  Current Score  \\\n",
+       "0  Ed-Fi-Exchange-OSS/LMS-Toolkit             5.9            5.3   \n",
+       "\n",
+       "   Percentage Change  \n",
+       "0         -10.169492  "
+      ]
+     },
+     "metadata": {},
+     "output_type": "display_data"
     }
    ],
    "source": [
-    "from datetime import datetime\n",
+    "from edfi_tech_metrics.ossf_score import compare_with_previous_scores\n",
     "\n",
-    "# Save the scores to a CSV file\n",
-    "now = datetime.strftime(datetime.now(), \"%Y-%m-%d\")\n",
-    "output_file = f\"./data/ossf-scores/{now}.csv\"\n",
-    "ossf_scores_df.to_csv(output_file, index=False)\n",
+    "comparison_df, improved_repos, worsened_repos = compare_with_previous_scores(ossf_scores)\n",
     "\n",
-    "print(f\"OSSF scores saved to {output_file}\")"
+    "# Display improved and worsened repositories\n",
+    "if improved_repos:\n",
+    "    improved_df = comparison_df[comparison_df['Repository'].isin(improved_repos)][['Repository', 'Previous Score', 'Current Score', 'Percentage Change']]\n",
+    "    improved_df = improved_df.sort_values('Percentage Change', ascending=False).reset_index(drop=True)\n",
+    "    print(\"\\nRepositories improved by 10% or more:\")\n",
+    "    display(improved_df)\n",
+    "else:\n",
+    "    print(\"\\nNo repositories improved by 10% or more.\")\n",
```

Comment:
Suggestion: [use markdown](https://stackoverflow.com/a/36313217/30384) to improve the display of this text.